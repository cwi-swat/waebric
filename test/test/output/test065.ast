module(module-id(["test065"]),[def("f",formals(["a","b"]),[echo-embedding(pre("\"A = <",exp-embedding([],var("a")),post(">\""))),echo-embedding(pre("\"B = <",exp-embedding([],var("b")),post(">\"")))]),def("main",empty,[markup(call(tag("f",[]),args([num(1)])))])])
