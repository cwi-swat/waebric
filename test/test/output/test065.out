<html>A = 1B = undef</html>

